src/libraries/Common/src/Interop/Windows/SspiCli/SecuritySafeHandles.cs

```diff
@@ -429,36 +429,31 @@ internal abstract partial class SafeDeleteContext : SafeHandle
 
             // Optional output buffer that may need to be freed.
             SafeFreeContextBuffer outFreeContextBuffer = null;
+            Span<IntPtr> ptr = stackalloc IntPtr[3];
```

Comment:
can `ptr` have a better name?

Reply:
Maybe pinnedTokenPointers, or something like that?

Reply:
I removed it completely based on feedback bellow.